posters-overrides.json

```diff
@@ -2373,7 +2373,7 @@
   },
   {
    "number": 851,
-   "pdf": "https://github.com/krobro/OHBM2020/raw/master/TGT-WMposter_OHBM2020.pdf"
+   "pdf": "https://raw.githubusercontent.com/krobro/OHBM2020/master/TGT-WMposter_OHBM2020.png"
```

Comment:
why not PDF which might have hyperlinks, author metadata, etc?

Reply:
I see we don't have preview for it but this wouldn't fix that -- we will likely to regenerate previews anyways

Reply:
Yes, sorry, my PI only provided the PNG. My previous pull request was a PDF just converted from the original PNG anyway, so no hyperlinks.

I noticed another poster was PNG and did have a preview. Will this work?

Reply:
it would but I wonder what problem you are trying to solve here?  PDF and PNG look identical to me (and pdfdiff which I ran on converted png)

Reply:
yes, correct, they are identical. Either one would do. Except, my most recent pull request aims to resolve:
1) the current URL to the PDF (hosted on github) prompts the user to download the file rather than just render it in browser. Providing a PNG was a quick workaround to the problem.

And my my hope is that a thumbnail will be auto generated? If not, please advise. Thanks!

Reply:
thumbnail -- we would need to regenerate them and it wouldn't matter pdf or png.

re "opening" -- oh, "good to know".  Is it the same PDF you would have from official ohbm site?  

Those are "smarter" in providing 

```
$> wget -S https://cdn-akamai.6connex.com/645/1827//OHBM_Poster_Final_BrittanyIntzandt_15919912427438430.pdf
...
  Content-Type: application/pdf
```
while dumb raw github:
```
  Content-Type: application/octet-stream
```
which is what I believe makes it happen to open in a window if your browser supports embedding pdf via external app.

@soichih - may be https://developer.github.com/v3/media/ hints you right away on how we could tune up our invocation for PDF to provide Content-Type?

Reply:
>  Is it the same PDF you would have from official ohbm site?
no. the pdf is just a converted png. i don't have the OHBM site pdf. anything else needed from me?

Reply:
About opening pdf file on browser, I believe the behavior is browser specific, and since gh-pages server is already setting the content-type properly, it's really up to the browser, and user settings to decide what happens when user clicks on the PDF link. 

On Chrome, there is a setting specifically for PDF and you have to make sure that the PDF file is not set to download automatically. See > https://apemobile.zendesk.com/hc/en-us/articles/115001569212-PDF-files-are-downloading-instead-of-displaying-in-web-browser-or-opening-in-a-new-tab

![image](https://user-images.githubusercontent.com/923896/86291772-80108200-bbbd-11ea-87ec-6d2c808ddfd1.png)

Reply:
Try opening one poster which comes from ohbm cdn, and another from GitHub . For me it differs in chrome, and I think chrome decides based on content type behind pdf url. Unrelated to gh-pages

Reply:
This 851 is one of those from GitHub

Reply:
Ah.. you are right.. It looks like raw.githubusercontent decided to set `X-Content-Type-Options: nosniff` .. so this means the browser has to examine the content of the file themselves (webserver won't help you decide which file type it is).

There is a service called `raw.githack.com` that is designed just to solve this problem..
> https://raw.githack.com/

We could replace all links to "https://raw.githubusercontent.com/something" with "https://raw.githack.com/something"
> https://raw.githack.com/krobro/OHBM2020/master/TGT-WMposter_OHBM2020.pdf
and that should solve the problem?

Reply:
Let's just do it on the fly may be in frontend?, otherwise PR submitters would need to know, would be pain to review etc

Reply:
I've added a code in the frontend to modify the *normal* github raw URL to raw.githack.com URL.

Reply:
#163 was merged and addresses this issue, so no changes needed, closing

Reply:
Nicely done!